pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "linkml-model-enrichment"
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
I would do this in a separate PR, only immediately prior to release...

but no biggie for now, we don't even have releases for this repo yet

Reply:
Yeah, that was an accidental mega merge. I have been asserting version because I'm pushing to PyPI now, for use in the NMDC DataHarmonizer. We should touch base on what method I use and what metadata you want to include in PyPI.